src/index/formatFileStructure.ts

```diff
@@ -7,55 +7,55 @@ import { fixImports } from "./formatFileStructure/fixImports";
 import { RootOption } from "./shared/RootOption";
 import logger from "../shared/logger";
 
+/** The main process when calling `destiny run [folder]` */
 export const formatFileStructure = async (
   rootDirFiles: string[][],
   filesToEdit: string[]
 ) => {
-  const unusedFiles: string[] = [];
   const rootOptions: RootOption[] = [];
+  const unusedFiles: string[] = [];
 
+  logger.info("Generating a graph and fractal tree.");
   for (const startingFiles of rootDirFiles) {
-    if (startingFiles.length <= 1) {
-      continue;
-    }
-
-    logger.info("Generating a graph and fractal tree.");
+    if (startingFiles.length <= 1) continue;
 
     const { graph, files, useForwardSlash, parentFolder } = buildGraph(
       startingFiles
     );
-    const tree = toFractalTree(graph, findEntryPoints(graph));
-    const usedFiles = new Set(Object.entries(graph).flat(2));
+    const entryPoints = findEntryPoints(graph);
+    const tree = toFractalTree(graph, entryPoints);
 
     rootOptions.push({
       tree,
       useForwardSlash,
       parentFolder,
     });
 
-    files.forEach(file => {
-      if (!usedFiles.has(file)) {
-        unusedFiles.push(file);
-      }
-    });
+    // This pattern allows skips over deeply nested paths,
+    // returning only paths that exist at this level.
```

Comment:
What?

Reply:
Relevant when using a `Set`. I'll get rid of this.

Reply:
When constructing a `Set`, if the passed in value is `[ 'value1', 'value 2', [ 'value3' ], 'value4']`, it will return a new Set containing `[ 'value1', 'value 2', 'value4']`.

Is there a better way to explain this?